### ISSUE TITLE
Feature: Use Background to avoid repeating the same steps in the subsequent scenarios

### ISSUE THREAD

**Issue by rhc** [olexale/bdd_widget_test#7]
I have a question related to the Background when you don't want to repeat the same statements at the beginning of each scenario

Example

```
Feature: Counter

  Scenario: starts with initial value of 0
    Given I visit the {'/counters'} page
    Then I see {"0"}


  Scenario:  displays 'Trick!' for the multiple of  3
    Given I visit the {'/counters'} page
    When I tap {Icons.add} icon {6} times
    Then I see {"6"}
    And I see {"Trick!"}
  #
```

Would it be possible to simply add a background?

```
Feature: Counter

  Background:
     Given I visit the counter page

  Scenario: starts with initial value of 0
    Given I visit the {'/counters'} page
    Then I see {"0"}


  Scenario:  displays 'Trick!' for the multiple of 3
    Given I visit the {'/counters'} page
    When I tap {Icons.add} icon {6} times
    Then I see {"6"}
    And I see {"Trick!"}
  #
```

**Comment by rhc** [comment]
Check also a similar effort in this package [flutter_test_ui 1.0.0](https://github.com/spkersten/flutter_test_ui)

**Comment by olexale** [comment]
Implemented in version [0.6.0](https://pub.dev/packages/bdd_widget_test), now this will work:
```feature
Feature: Counter

  Background:
    Given I visit the {'/counters'} page

  Scenario: starts with initial value of 0
    Then I see {"0"}

  Scenario:  displays 'Trick!' for the multiple of 3
    When I tap {Icons.add} icon {6} times
    Then I see {"6"}
    And I see {"Trick!"}
```

**Comment by rhc** [comment]
Thanks Ole!
